### PR TITLE
[SD-2825] Fixed notification for granting privileges to a role.

### PR DESCRIPTION
**Pull request by mdhaman**

### CODE REVIEW

File: superdesk/roles/roles.py [hunk 1]
```diff
@@ -93,11 +93,12 @@ def __send_notification(self, updates, role):
...
-                             'role {{role}} is granted new privileges: Please re-login.',
-                             self.datasource,
-                             notify=notified_users,
-                             role=role.get('name'))
+            if len(added) > 0 or (0, 1) in modified.values():
+                activity = add_activity(ACTIVITY_UPDATE,
+                                        'role {{role}} is granted new privileges: Please re-login.',
+                                        self.datasource,
+                                        notify=notified_users,
+                                        role=role.get('name'))
```
Comment (sivakuna-aap) [inline superdesk/roles/roles.py]: Do we need 2 push notifications? If not can we do the same thing what we are doing in desks.py?

**Comment by sivakuna-aap** [comment]
:+1: except the minor one

File: superdesk/roles/roles.py (hunk 1 above)
Comment (mdhaman) [inline superdesk/roles/roles.py]: not required.

**Comment by marwoodandrew** [comment]
:+1:

**Pull request merged**